### ISSUE TITLE
odo link don't understand --component flag

### ISSUE THREAD

**Issue by jorgemoralespou** [redhat-developer/odo#928]
[kind/bug]

## What versions of software are you using?

Operating System : macOS 10.14

Output of `odo version` : 
```
odo v0.0.14 (da250f3)

Server: xxx
OpenShift: v3.11.16
Kubernetes: v1.11.0+d4cacc0
```

## How did you run odo exactly?

```
$ odo link nationalparks-db --component nationalparks
Error: unknown flag: --component
Usage:
  odo link <service> --component [component] [flags]

Examples:
  # Link the current component to the 'my-postgresql' service
  odo link my-postgresql

  # Link component 'nodejs' to the 'my-postgresql' service
  odo link my-postgresql --component nodejs


Flags:
      --app string       Application, defaults to active application
  -h, --help             help for link
      --project string   Project, defaults to active project
  -w, --wait             If enabled, the link command will wait for the service to be provisioned if it's not yet

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files (default false)
      --skip-connection-check            Skip cluster check
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs. Level varies from 0 to 9 (default 0).
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

unknown flag: --component
```

**Comment by jorgemoralespou** [comment]
cc/ @geoand

**Comment by kadel** [comment]
fixed in #932
